packages/canon/src/components/Tooltip/Tooltip.styles.css

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+.canon-TooltipPopup {
+  box-sizing: border-box;
+  font-size: 0.875rem;
+  line-height: 1.25rem;
+  display: flex;
+  flex-direction: column;
+  padding: 0.25rem 0.5rem;
+  border-radius: 0.375rem;
+  background-color: canvas;
+  transform-origin: var(--transform-origin);
+  transition: transform 150ms, opacity 150ms;
+  background-color: var(--canon-bg-surface-1);
+  color: var(--canon-fg-primary);
+  outline: 1px solid var(--canon-border);
+  box-shadow: 0 10px 15px -3px var(--canon-border),
+    0 4px 6px -4px var(--canon-border);
+
+  &[data-starting-style],
+  &[data-ending-style] {
+    opacity: 0;
+    transform: scale(0.9);
+  }
+
+  &[data-instant] {
+    transition-duration: 0ms;
+  }
+}
+
+.canon-TooltipArrow {
+  display: flex;
+
+  &[data-side='top'] {
+    bottom: -8px;
+    rotate: 180deg;
+  }
+
+  &[data-side='bottom'] {
+    top: -8px;
+    rotate: 0deg;
+  }
+
+  &[data-side='left'] {
+    right: -13px;
+    rotate: 90deg;
+  }
+
+  &[data-side='right'] {
+    left: -13px;
+    rotate: -90deg;
+  }
+}
+
+.canon-TooltipArrowFill {
+  fill: var(--canon-bg-surface-1);
+}
+
+.canon-TooltipArrowOuterStroke {
+  @media (prefers-color-scheme: light) {
+    fill: var(--canon-border);
+  }
+}
+
+.canon-TooltipArrowInnerStroke {
```

Comment:
feeling still it might be good to have a divider to tell apart the component name from the rest in the class

Reply:
Ah you're right. I'll change that